Autopilot/Inc/ultrasonic.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Ultrasonic Sensor Functions
+ * Author: Dhruv Upadhyay, February 2021
+ */
+
+ #ifndef ULTRASONIC_HPP
+ #define ULTRASONIC_HPP
+
+ #include <stdint.h>
```

Comment:
Wait do you need stdint.h?

Reply:
I wanted to make an int with fix width, so that's why I included stdint as well. Is that not necessary in the header?